test_utils/os_utils.py

```diff
@@ -379,10 +380,13 @@ def get_udev_service_path(unit_name):
     return path
 
 
-def kill_all_io():
-    # TERM signal should be used in preference to the KILL signal, since a
-    # process may install a handler for the TERM signal in order to perform
-    # clean-up steps before terminating in an orderly fashion.
+def kill_all_io(graceful=True):
+    if graceful:
+        # TERM signal should be used in preference to the KILL signal, since a
+        # process may install a handler for the TERM signal in order to perform
+        # clean-up steps before terminating in an orderly fashion.
+        TestRun.executor.run("killall -q --signal TERM dd fio blktrace")
+        time.sleep(3)
```

Comment:
so... gracefulness is just trying the nice way one more time before unleashing hell? ;)

Reply:
+